docs/content/preview/contribute/core-database/run-unit-tests.md

```diff
@@ -44,13 +44,13 @@ To run a specific sub-test, for example the `TestMonoTime.TestCondition` sub-tes
 
 ### Run all tests
 
-Given that you've already built C++ and Java code you can run Java tests using following command:
+After you've already built the C++ and Java code, you can run the Java tests using following command:
 
 ```sh
 ./yb_build.sh release --scb --sj --java-tests
 ```
 
-If you omit release argument, it will run java tests against debug YugabyteDB build, so you should then either build debug binaries with `./yb_build.sh` or omit `--scb` and then it will build debug binaries automatically.
+If you omit the release argument, it will run java tests against debug build, so you should either omit `--scb` or first build debug binaries by running `./yb_build.sh` with no test options.
```

Comment:
```suggestion
If you omit the release argument, it will run Java tests against the debug build, so you should either omit `--scb` or first build debug binaries by running `./yb_build.sh` with no test options.
```